### PR TITLE
Replay `3318494`: `ReturnSection` precondition failure

### LINKED ISSUE #1145: Precondition Check Fails in `ReturnSection`
## Ranges

```rust
3318494-3318494
```

## Issue

For block `3318494` on Sepolia, we get the following error:

```
ERROR net.consensys.linea.zktracer.ZkTracer -- First exception that was caught while tracing:
java.lang.IllegalArgumentException: null
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:129)
	at net.consensys.linea.zktracer.module.hub.section.halt.ReturnSection.<init>(ReturnSection.java:229)
	at net.consensys.linea.zktracer.module.hub.Hub.traceOpcode(Hub.java:1078)
	at net.consensys.linea.zktracer.module.hub.Hub.processStateExec(Hub.java:1001)
	at net.consensys.linea.zktracer.module.hub.Hub.tracePreExecution(Hub.java:774)
	at net.consensys.linea.zktracer.ZkTracer.tracePreExecution(ZkTracer.java:258)
	at net.consensys.linea.zktracer.ConflationAwareOperationTracer$SequencingOperationTracer.lambda$tracePreExecution$5(ConflationAwareOperationTracer.java:97)
	at java.base/java.util.Arrays$ArrayList.forEach(Arrays.java:4305)
	at net.consensys.linea.zktracer.ConflationAwareOperationTracer$SequencingOperationTracer.tracePreExecution(ConflationAwareOperationTracer.java:97)
	at org.hyperledger.besu.evm.EVM.runToHalt(EVM.java:207)
```

I have created a test for this, which is to be merged as part of #1143 (marked as `@Disabled`)

```
  @Test
  void largeInitCode() {
    replay(LINEA_SEPOLIA, "3318494.sepolia.json.gz", true);
  }
```

**NOTE:** #1143 fixes problems related to the replay file not replaying correctly.

### PR DESCRIPTION
**Pull request by OlivierBBB**
## Block

```rust
3318494-3318494
```

## Issue

Precondition failure in the `ReturnSection`'s
```java
      checkArgument(!maxCodeSizeOobCall.isMaxCodeSizeException());
```
Upon inspection the code size was $\mathtt{0x\\,4xxx} \leq \mathtt{0x\\,6000}$ so no $\mathtt{maxCodeSizeException}$ should have been thrown in the first place.

### CODE REVIEW

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java [hunk 1]
```diff
@@ -288,7 +288,7 @@ private void populateColumnsForEvmInstruction(MessageFrame frame) {
       }
       case OOB_INST_DEPLOYMENT -> {
         final DeploymentOobCall deploymentOobCall = (DeploymentOobCall) oobCall;
-        deploymentOobCall.setSize(EWord.of(frame.getStackItem(0)));
+        deploymentOobCall.setSize(EWord.of(frame.getStackItem(1)));
```
Comment (OlivierBBB) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java:291]: Only important change: the `size` is in the 1st position.

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java [hunk 2]
```diff
@@ -542,10 +542,11 @@ private BigInteger callToMOD(
 
   private boolean callToLT(
       int k, BigInteger arg1Hi, BigInteger arg1Lo, BigInteger arg2Hi, BigInteger arg2Lo) {
-    checkArgument(arg1Hi.bitLength() / 8 <= 16);
-    checkArgument(arg1Lo.bitLength() / 8 <= 16);
-    checkArgument(arg2Hi.bitLength() / 8 <= 16);
-    checkArgument(arg2Lo.bitLength() / 8 <= 16);
+    // 128 = 8 * 16
+    checkArgument(arg1Hi.bitLength() <= 128);
```
Comment (letypequividelespoubelles) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java:546]: I'd prefer to have a `LIMB_SIZE` constants in GlobalConstants.java, (so from the `constant/constants.lisp`)

Comment (letypequividelespoubelles) [reply arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java:546]: I mean, one `LIMB_BYTE_SIZE` and one `LIMB_BIT_SIZE` to be explicit

**Review by letypequividelespoubelles — APPROVED** [review]
LGTM

**Pull request merged**